bgpd/bgp_route.c

```diff
@@ -2310,6 +2310,13 @@ static void bgp_process_main_one(struct bgp *bgp, struct bgp_node *rn,
 	char pfx_buf[PREFIX2STR_BUFFER];
 	int debug = 0;
 
+	if (bgp_flag_check(bgp, BGP_FLAG_DELETE_IN_PROGRESS)) {
+		if (debug) {
```

Comment:
This is always false.

Reply:
This allows someone to come and change debug to on when debugging.

Reply:
I don't think that justifies adding dead code.